browser/mapping_test.go

```diff
@@ -17,6 +17,18 @@ import (
 	k6metrics "go.k6.io/k6/metrics"
 )
 
+// wildcards is a list of extra mappings for our API (api/).
```

Comment:
It should be harmless moving these here. On the other hand, having these on the `mapping.go` file's top location is to demonstrate to someone who might be reading the code which custom mappings we have—They can learn by reading the tests too—so it's harmless to move them here. Just a note :)